.gitignore

```diff
@@ -9,6 +9,7 @@ geopandas/version.py
 doc/_static/world_*
 examples/nybb_*.zip
 .cache/
+*.c
```

Comment:
But that is added, so then it is no problem? 
I thought this just means that you have to force add new c files, but maybe that's an incorrect git understanding